Core/AppDeepLinks.swift

```diff
@@ -39,4 +41,8 @@ public struct AppDeepLinks {
         }
         return false
     }
+    
+    public static func query(fromQuickLink url: URL) -> String {
+        return url.absoluteString.replacingOccurrences(of: quickLink, with: "")
```

Comment:
Yeah I had trouble naming this one! It's a link that launches straight into the browser for example an external link (bookmark/favourite) or an internal link outside the browsing context (as in this case where we launch the privacy url from settings).